cmd/renterd/main.go

```diff
@@ -351,8 +351,10 @@ func main() {
 
 	network, _ := build.Network()
 	busCfg := node.BusConfig{
-		Bus:     cfg.Bus,
-		Network: network,
+		Bus:                 cfg.Bus,
+		Network:             network,
+		SlabPruningInterval: time.Hour,
+		SlabPruningCooldown: 30 * time.Second,
```

Comment:
seems very low, I would increase it to roughly the sum of the retry tx intervals?

Reply:
The largest txn retry interval is 10 seconds. I think this is a reasonable tradeoff between UI lag and improved performance for now. We can always increase it or lower it again.